### PR TITLE
add datefinder python package

### PR DESCRIPTION
**Pull request by melonhead901**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/datefinder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: acab2bebfdb09fdc9e9d512b280bf676a9a03f286c666234d50cfe65a5da7f7d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
```
Comment (dopplershift) [inline recipes/datefinder/meta.yaml:19]: You need to add `pip` to your host requirements if you're going to use `pip` above to build the package.

File: recipes/datefinder/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  url: "https://github.com/akoumjian/datefinder/archive/v0.7.1.tar.gz"
+  sha256: acab2bebfdb09fdc9e9d512b280bf676a9a03f286c666234d50cfe65a5da7f7d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/datefinder/meta.yaml:18]: The host section probably only needs `python` and `pip`

File: recipes/datefinder/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{% set name = "datefinder" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/akoumjian/datefinder/archive/v0.7.1.tar.gz"
```
Comment (synapticarbors) [inline recipes/datefinder/meta.yaml:9]: `url: "https://github.com/akoumjian/datefinder/archive/v{{ version}}.tar.gz"`

Also in the future if https://github.com/akoumjian/datefinder/pull/130 gets merged, you can use the pypi release without having to manually package the license file.

File: recipes/datefinder/meta.yaml (hunk 1 above)
Comment (melonhead901) [reply recipes/datefinder/meta.yaml:19]: Done

File: recipes/datefinder/meta.yaml (hunk 2 above)
Comment (melonhead901) [reply recipes/datefinder/meta.yaml:18]: Done

File: recipes/datefinder/meta.yaml (hunk 3 above)
Comment (melonhead901) [reply recipes/datefinder/meta.yaml]: Done -- ACK on the LICENSE

**Comment by synapticarbors** [comment]
@melonhead901 -- thank you for your contribution to conda-forge. If you run into any issues once the feedstock is created, don't hesitate to reach out.

**Pull request merged**